zio-http/src/main/scala/zhttp/http/Path.scala

```diff
@@ -36,6 +36,10 @@ final case class Path(segments: Vector[String], trailingSlash: Boolean) { self =
 
   def toList: List[String] = segments.toList
 
+  def dropTrailingSlash: Path = self.copy(trailingSlash = false)
+
+  def addTrailingSlash: Path = self.copy(trailingSlash = true)
+
```

Comment:
Need this API in `Url` also as mentioned here — https://github.com/dream11/zio-http/pull/1291#pullrequestreview-1002306902